scripts/lib/CIME/XML/env_batch.py

```diff
@@ -256,7 +255,7 @@ def get_submit_args(self, case, job):
             else:
                 if name.startswith("$"):
                     name = name[1:]
-                val = case.get_value(name, subgroup=job)
+                val = case.get_value(name,subgroup=job)
```

Comment:
Please ignore this diff; it's one I missed when copying from the ACME branch to CIME